security/js-leaks.html

```diff
@@ -29,6 +29,7 @@
         <option value="chrome_110">Chrome 110</option>
         <option value="safari_14">Safari 14</option>
         <option value="safari_14_ios">Safari iOS 14</option>
+        <option value="safari_16_1">Safari 16.1</option>
```

Comment:
Nit - I think we should only keep the latest version and remove the old ones (safari 14). I don't think there is any use for them?